### PR TITLE
Added unit tests for action_service, configuration and device_status

### PR DESCRIPTION
**Pull request by animeshjaipurkar**
Added unit tests for action_service.py, configuration.py and device_status.py

### CODE REVIEW

File: tests/test_action_service.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+@patch('bot_python_sdk.configuration.Configuration.get_device_id')
+@patch('bot_python_sdk.key_generator.KeyGenerator.generate_uuid')
+def test_create_trigger_body(generate_uuid, get_device_id):
+
+	as_obj = ActionService()
+
+	uuid = 42
+	generate_uuid.return_value = uuid
+	device_id = 43
```
Comment (ercanbozogluing) [inline tests/test_action_service.py]: The device_id should be a string value ('43')

Comment (animeshjaipurkar) [reply tests/test_action_service.py]: Thank you. I have now updated the device_id to a string value.

**Comment by ndpatnaik** [comment]
Changes are ok. Can you please remove a special character being seen at the end of the file "tests/test_configuration.py"

**Comment by animeshjaipurkar** [comment]
> Changes are ok. Can you please remove a special character being seen at the end of the file "tests/test_configuration.py"

Thank you. I have now removed the special character at the end of test_configuration.py

**Comment by animeshjaipurkar** [comment]
> @animeshjaipurkar you can merge the PR :)

Thank you, but I don't have the access to merge pull request.  I can see the following message: 
_"Only those with write access to this repository can merge pull requests."_

**Comment by nanchauha** [comment]
It's merged. Thanks for the PR @animeshjaipurkar

**Comment by animeshjaipurkar** [comment]
@nanchauha Thank you :)

**Pull request merged**